packages/core/src/constructor.ts

```diff
@@ -237,7 +237,7 @@ export function instantiateInteractor<E extends Element, F extends Filters<E>, A
       Object.defineProperty(interactor, filterName, {
         value: function() {
           return interactionFilter(`${filterName} of ${description(options)}`, async () => {
-            return applyFilter(filter, resolver(options));
+            return applyFilter(filter, await converge(() => resolver(options) ));
```

Comment:
```suggestion
            return converge(() => applyFilter(filter,  resolver(options)));
```

Reply:
We don't want to release the event loop in between when we resolve the element and apply the filter.